### PR TITLE
feat: transition docs

### LINKED ISSUE #1468: [FEATURE] information about available transitions in docs
**Describe how this feature will help us improve**
There is no information about available transitions in our docs. We need to add this

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue
Closes #1468 

# Scope of work
docs for transitions 

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/utilities/transitions/transitions.stories.js [hunk 1]
```diff
@@ -1,5 +1,68 @@
 import SfButton from "./../../../src/components/atoms/SfButton/SfButton.vue";
 
+const doscTransitions = `<div>
```
Comment (mayashavin) [inline packages/vue/src/utilities/transitions/transitions.stories.js:3]: If there is no Vue Component involved, use MDX instead of this :)

**Review by mayashavin — CHANGES_REQUESTED** [review]
See comments

File: packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Releases/v0.11.0 - Latest/Change log" />
+
+# v0.11.0
+
+## What is new?
+
+👉 docs for transitions in `Utilites/Transitions/Docs`
```
Comment (mayashavin) [inline packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx:9]: Please follow our standard of writing in v.10.0 release note.

**Review by mayashavin — CHANGES_REQUESTED** [review]

File: packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx (hunk 2 above)
Comment (justyna-13) [reply packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx]: done :)

File: packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Meta } from "@storybook/addon-docs/blocks";
 
-<Meta title="Releases/v0.10.1 - Latest/Change log" />
```
Comment (mayashavin) [inline packages/vue/src/stories/releases/v0.10.1/v0.10.1.stories.mdx:3]: Please revert the change here.

File: packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from "@storybook/addon-docs/blocks";
```
Comment (mayashavin) [inline packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx:1]: I think we don't need this file yet

**Review by mayashavin — CHANGES_REQUESTED** [review]
See my comments. I will change the branch to merge directly to `master` since this is docs-related.

File: packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx (hunk 4 above)
Comment (mayashavin) [reply packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx:1]: Please change this file to 0.10.2

**Review by mayashavin — APPROVED** [review]
